src/__tests__/strictmode.test.ts

```diff
@@ -80,7 +80,7 @@ test('Strict mode should unsubscribe the source observable on unmount if its cre
   }
 
   const {rerender} = render(createElement(StrictMode, null, createElement(ObservableComponent)))
-  expect(subscriberCount, 'Subscriber count should be 2').toBe(2)
+  expect(subscriberCount, 'Subscriber count should be 1').toBe(1)
```

Comment:
In React 19 StrictMode will re-use the cache from `useMemo` on the second render, so the subscribe count is now 1 instead of 2: https://react.dev/blog/2024/04/25/react-19-upgrade-guide#strict-mode-improvements

cc @bjoerge